cpp/oneapi/dal/table/backend/csr_table_impl.hpp

```diff
@@ -29,46 +29,59 @@ class csr_table_impl : public detail::csr_table_template<csr_table_impl>,
 public:
     csr_table_impl()
             : col_count_(0),
-              row_count_(0),
               layout_(data_layout::row_major),
-              csr_indexing_(detail::csr_indexing::one_based) {}
+              indexing_(sparse_indexing::one_based) {}
```

Comment:
Just curious, why is one_based the default here (and throughout)?

Reply:
I've chosen `one_based` as a default for compatibility with legacy DAAL interface which supports only one-based indexing. In order not to have extra copies when passing the data between oneDAL and DAAL.